FEXCore/Scripts/json_ir_generator.py

```diff
@@ -641,7 +641,7 @@ def print_ir_allocator_helpers():
     output_file.write("\t}\n\n")
 
     output_file.write("\tuint8_t GetOpElements(const OrderedNode *Op) const {\n")
-    output_file.write("\t\tauto HeaderOp = Op->Header.Value.GetNode(DualListData.DataBegin());\n")
+    output_file.write("\t\t[[maybe_unused]] auto HeaderOp = Op->Header.Value.GetNode(DualListData.DataBegin());\n")
     output_file.write("\t\tLOGMAN_THROW_A_FMT(OpHasDest(Op), \"Op {} has no dest\\n\", GetName(HeaderOp->Op));\n")
```

Comment:
Could also delete the `HeaderOp` definition in the previous line and use `GetOpName(Op)` inside the assertion.

Reply:
Good point - done.